ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -80,6 +80,7 @@ Login To RHODS Dashboard
    IF  ${login-required}  Login To Openshift  ${ocp_user_name}  ${ocp_user_pw}  ${ocp_user_auth_type}
    ${authorize_service_account}=  Is rhods-dashboard Service Account Authorization Required
    IF  ${authorize_service_account}  Authorize rhods-dashboard service account
+   Navigate To Page    Applications    Enabled
```

Comment:
hm as far as I remember, tests use `Wait For RHODS Dashboard To Load` which allows you to set the expected page. So the tests don't wait explicitly for "Enabled" page.

Reply:
> set the expected page

Reply:
default can be changed

Reply:
Later, yes. And I prefer not to have a default value.

Reply:
So... is the time to revert this finally?

Reply:
Suppose you are going to change something related to this code. Please make sure you do it explicitly.
In this case, if we read the code we know exactly what is happening.

Reply:
Yeah, I bumped into this when reviewing failures of some of our tests. I have two options basically:
1. provide a bit ugly and easy solution (at least from my point of view) to change a couple of our tests only
2. revert this workaround and check that no tests are broken which is much more work - but seem like a proper solution for me

And I would prefer to avoid both of the above of course... :upside_down_face:

Reply:
FTR #1725